src/test/java/integration/fakeevents/controllers/RoomControllerIntTest.java

```diff
@@ -0,0 +1,39 @@
+package integration.fakeevents.controllers;
+
+import integration.fakeevents.BaseIntegrationTestWithFakeEvents;
+import io.restassured.http.Header;
+import org.junit.Test;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.getRequestedFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+import static com.github.tomakehurst.wiremock.client.WireMock.verify;
+import static io.restassured.RestAssured.given;
+
+public class RoomControllerIntTest extends BaseIntegrationTestWithFakeEvents {
+
+    @Test
+    public void rooms_shouldCallProperEventsEndpointAndReturnRooms() throws Exception {
+        stubFor(get(urlEqualTo("/room"))
+            .willReturn(aResponse()
+                .withStatus(200)
+                .withHeader("Content-Type", "application/json")
+                .withBody("[\"rooms\"]")));
+
+        Header authenticationHeader = signIn("officer1", "asd");
```

Comment:
I'm pretty sure this could be standardized somewhere in a 'helper' method as it does occur (almost) always.

Reply:
As it's jwt we could probably prepare a token with expiry like 2300 and treat it as a constant.